inc/utils/content.php

```diff
@@ -564,17 +581,19 @@ function get_categories_array($post){
 
  function odm_echo_extras($postID = "") {
  	 $postID = $postID ? $postID : get_the_ID();
-	 if (function_exists('get_post_meta')) :
-		 $get_author = get_post_meta($postID, 'author', true);
-		 $get_localized_author = get_post_meta($postID, 'author_'.odm_language_manager()->get_current_language(), true);
-	   if ($get_author != '' || $get_localized_author != ''):
-	     $news_source_info = '<span class="lsf">&#xE041;</span> ';
-	     if ($get_localized_author != ''):
-	         $news_source_info .= $get_localized_author.'<br />';
-	     else:
-	         $news_source_info .= $get_author.'<br />';
-	     endif;
-	   endif;
+	 if (function_exists('get_post_meta')):
```

Comment:
@acorbi, the idea is that the author should show inline with the other meta tags and category for all the post types, except the news articles. 
Since, the news contains both author or original link of the source, so editor suggests to keep the author next to the link, otherwise, it doesn't make sense. Moreover, ODC would like to highlight the citation of the news source to avoid any problem that used to happened in the future.